tests/libvmi/factory.go

```diff
@@ -77,7 +77,7 @@ func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(alpineOpts...)
 }
 
-func cirrosMemory() string {
+func CirrosMemory() string {
```

Comment:
I would prefer to think twice before exposing this function. Ideally, it is an implementation detail of `NewCirros`. I you have to do so, please explain why in a clear commit message.